src/main/java/org/elasticsearch/search/highlight/PlainHighlighter.java

```diff
@@ -164,6 +164,11 @@ public int compare(TextFragment o1, TextFragment o2) {
         return null;
     }
 
+    @Override
+    public boolean cannotHighlight(FieldMapper fieldMapper) {
+        return false;
```

Comment:
as said above, the double negation here feels weird to me, but again maybe it's just me